platform/fabric/services/chaincode/endorse.go

```diff
@@ -65,11 +65,7 @@ func (i *endorseChaincodeView) Endorse(context view.Context) (*fabric.Envelope,
 		i.SignerIdentity = fNetwork.IdentityProvider().DefaultIdentity()
 	}
 
-	var chaincode Chaincode
-	chaincode = &stdChaincode{ch: channel.Chaincode(i.ChaincodeName)}
-	if chaincode == nil {
-		return nil, errors.Errorf("fabric chaincode [%s:%s:%s] not found", i.Network, i.Channel, i.ChaincodeName)
-	}
+	var chaincode Chaincode = &stdChaincode{ch: channel.Chaincode(i.ChaincodeName)}
```

Comment:
Can't we do the following?
```suggestion
	chaincode := &stdChaincode{ch: channel.Chaincode(i.ChaincodeName)}
```